### ISSUE TITLE
ui5-textarea: aria-invalid not announced

### ISSUE THREAD

**Issue by niyap** [SAP/ui5-webcomponents#4766]
### **Bug Description**
Like in the ui5-input component, the aria-invalid attribute within ui5-textarea should be added to the textarea HTML element instead of being on the root element. Something more, in order to be announced that the ui5-textarea is invalid when value state error is added, the aria-invalid attribute should have explicitly added value equals to "true".
